examples/demo-rollup/rollup_config.toml

```diff
@@ -13,3 +13,9 @@ max_celestia_response_body_size = 104_857_600
 [runner.storage]
 # The path to the rollup's data directory. Paths that do not begin with `/` are interpreted as relative paths.
 path = "demo_data"
+
+[rpc_config]
+# the host and port to bind the rpc server for
+bind_host = "127.0.0.1"
+bind_port = 12345
```

Comment:
Is this the port Celestia currently exposes by default? Would be nice if users didn't need to change it.

Reply:
oh this is the configuration for the RPC that that full node would run, so the user can set whichever port and bind to whichever network interface they want